src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/013_117_revocaDelegaPG.feature

```diff
@@ -15,10 +15,6 @@ Feature: La persona giuridica revoca una delega
   @fase2Test117
   @new
   Scenario: La persona giuridica revoca una delega
-    When Nella pagina Piattaforma Notifiche persona giuridica click sul bottone Deleghe
-    And Si visualizza correttamente la pagina Deleghe sezione Deleghe a Carico dell impresa
-    And Nella pagina Deleghe si clicca su Delegati dall impresa
-    And Si visualizza correttamente la pagina Deleghe sezione Deleghe dell impresa
```

Comment:
come prima

Reply:
sono stati tutti eliminati perchè i stessi step venivano fatti nel featur in alto